packages/esbuild-plugin-html/lib/index.js

```diff
@@ -98,6 +98,25 @@ export default function({ scriptsTarget = 'es2015', modulesTarget = 'es2020' } =
                         await writeFile(outputFile, buffer);
                         outputFile = path.relative(process.cwd(), outputFile);
                         outputFiles.push(outputFile);
+                        addBundleMetafile({
```

Comment:
Why not collect the whole metafile and then pass it to the callback?

Reply:
Is this what you meant?